### PR TITLE
Update documentation for macOS, remove references to Sample Web UI that no longer exists

### PR DESCRIPTION
**Pull request by amorde**
I'm new to Kythe and got tripped up a bit by the documentation being a bit out of date.

### CODE REVIEW

File: kythe/web/site/getting-started-macos.md
Comment (amorde) [inline kythe/web/site/getting-started-macos.md:73]: Example error encountered when using the built-in macOS `sed`:

```
ERROR: /Users/eamorde/Development/kythe/kythe/java/com/google/devtools/kythe/extractors/java/bazel/BUILD:6:12: 
//kythe/java/com/google/devtools/kythe/extractors/java/bazel:java_extractor depends on 
@io_bazel_files//src/java_tools/buildjar/java/com/google/devtools/build/buildjar:javac_options in repository 
@io_bazel_files which failed to fetch. no such package 
'@io_bazel_files//src/java_tools/buildjar/java/com/google/devtools/build/buildjar': sed: illegal option -- -
usage: sed script [-Ealnru] [-i extension] [file ...]
	sed [-Ealnu] [-i extension] [-e script] ... [-f script_file] ... [file ...]
ERROR: Analysis of target '//kythe/extractors:extract_kzip_java' failed; build aborted:
```

File: kythe/web/site/getting-started-macos.md
Comment (amorde) [inline kythe/web/site/getting-started-macos.md:70]: the default homebrew path is now `/opt/homebrew/opt`. using `brew --prefix --installed [FORMULA]` gives you the prefix to the installed tool and will fail if the tool isn't installed by Homebrew

**Comment by salguarnieri** [comment]
Thanks for the updates.

The linter is failing some format checks. Can you follow https://kythe.io/contributing/#linting-and-testing so the files get formatted correctly? I think clang-format in particular will change some files once you set up the commit hooks.

**Comment by amorde** [comment]
I can't seem to install the pre-commit hooks because I have `core.hooksPath` set on my machine. Is there away for me to manually run the checks? Could you share the results of the lint job showing the lint failures?

**Comment by amorde** [comment]
It also seems like some of these fail to run on macOS

```
./tools/git/commit-msg.sh                                                                                
Error: Cannot find module "@commitlint/config-conventional" from "/Users/eamorde/Development/kythe"
    at resolveId (/private/var/tmp/_bazel_eamorde/7396ce93e4a02c32055c37ee57a28b26/execroot/io_kythe/bazel-out/darwin_arm64-fastbuild/bin/tools/git/commitlint.sh.runfiles/io_kythe/node_modules/.aspect_rules_js/@commitlint+resolve-extends@17.6.7/node_modules/@commitlint/resolve-extends/src/index.ts:131:14)
    at resolveConfig (/private/var/tmp/_bazel_eamorde/7396ce93e4a02c32055c37ee57a28b26/execroot/io_kythe/bazel-out/darwin_arm64-fastbuild/bin/tools/git/commitlint.sh.runfiles/io_kythe/node_modules/.aspect_rules_js/@commitlint+resolve-extends@17.6.7/node_modules/@commitlint/resolve-extends/src/index.ts:105:20)
    at /private/var/tmp/_bazel_eamorde/7396ce93e4a02c32055c37ee57a28b26/execroot/io_kythe/bazel-out/darwin_arm64-fastbuild/bin/tools/git/commitlint.sh.runfiles/io_kythe/node_modules/.aspect_rules_js/@commitlint+resolve-extends@17.6.7/node_modules/@commitlint/resolve-extends/src/index.ts:51:20
    at Array.reduce (<anonymous>)
    at loadExtends (/private/var/tmp/_bazel_eamorde/7396ce93e4a02c32055c37ee57a28b26/execroot/io_kythe/bazel-out/darwin_arm64-fastbuild/bin/tools/git/commitlint.sh.runfiles/io_kythe/node_modules/.aspect_rules_js/@commitlint+resolve-extends@17.6.7/node_modules/@commitlint/resolve-extends/src/index.ts:49:13)
    at resolveExtends (/private/var/tmp/_bazel_eamorde/7396ce93e4a02c32055c37ee57a28b26/execroot/io_kythe/bazel-out/darwin_arm64-fastbuild/bin/tools/git/commitlint.sh.runfiles/io_kythe/node_modules/.aspect_rules_js/@commitlint+resolve-extends@17.6.7/node_modules/@commitlint/resolve-extends/src/index.ts:25:19)
    at load (/private/var/tmp/_bazel_eamorde/7396ce93e4a02c32055c37ee57a28b26/execroot/io_kythe/bazel-out/darwin_arm64-fastbuild/bin/tools/git/commitlint.sh.runfiles/io_kythe/node_modules/.aspect_rules_js/@commitlint+load@17.7.2/node_modules/@commitlint/load/src/load.ts:56:33)
    at async main (/private/var/tmp/_bazel_eamorde/7396ce93e4a02c32055c37ee57a28b26/execroot/io_kythe/bazel-out/darwin_arm64-fastbuild/bin/tools/git/commitlint.sh.runfiles/io_kythe/node_modules/.aspect_rules_js/@commitlint+cli@17.7.2/node_modules/@commitlint/cli/src/cli.ts:214:17) {
  code: 'MODULE_NOT_FOUND'
}
```

**Comment by amorde** [comment]
Feel free to commit changed to my branch. Maintainers should have write access to my fork

File: .commitlintrc.js
Comment (shahms) [inline .commitlintrc.js:28]: The lint failure is because it wants to format this file because this line is too long. I think just replace "cross-reference" with "xref" and you'll be fine.

File: kythe/docs/kythes-command-line-tool.txt
Comment (shahms) [inline kythe/docs/kythes-command-line-tool.txt:22]: Remove the trailing whitespace here

File: kythe/web/site/examples.md
Comment (shahms) [inline kythe/web/site/examples.md:370]: And remove the trailing whitespace here.

**Comment by amorde** [comment]
What's the current lint failure?

**Comment by jaysachs** [comment]
Looks like commitlintrc.js
```
Step #1: diff --git a/.commitlintrc.js b/.commitlintrc.js
Step #1: index 00e078c0a..03750f60e 100644
Step #1: --- a/.commitlintrc.js
Step #1: +++ b/.commitlintrc.js
Step #1: @@ -20,57 +20,60 @@ module.exports = {
Step #1:    rules : {
Step #1:      'scope-case' : [ 2, 'always', 'snake-case' ],
Step #1:      'scope-empty' : [ 1, 'never' ],
Step #1: -    'scope-enum' : [
Step #1: -      1, 'always',
Step #1: -      [
Step #1: -        'api',             // user-facing API (e.g. XRefService)
Step #1: -        'dev',             // developer tooling/scripting (e.g. linting)
Step #1: -        'example',         // example tools or docs (e.g. sample xref service)
Step #1: -        'extraction',      // compilation extraction
Step #1: -        'indexing',        // semantic analysis over compilations
Step #1: -        'post_processing', // serving data construction
Step #1: -        'schema',          // graph schema design
Step #1: -        'serving',         // server logic or data formats
Step #1: -        'tooling',         // CLI utilities (e.g. entrystream)
Step #1: +    'scope-enum' :
Step #1: +                 [
Step #1: +                   1, 'always',
Step #1: +                   [
Step #1: +                     'api',        // user-facing API (e.g. XRefService)
Step #1: +                     'dev',        // developer tooling/scripting (e.g. linting)
Step #1: +                     'example',    // example tools or docs (e.g. sample xref
Step #1: +                                   // service)
Step #1: +                     'extraction', // compilation extraction
Step #1: +                     'indexing',   // semantic analysis over compilations
Step #1: +                     'post_processing', // serving data construction
Step #1: +                     'schema',          // graph schema design
Step #1: +                     'serving',         // server logic or data formats
Step #1: +                     'tooling',         // CLI utilities (e.g. entrystream)
Step #1:  
Step #1: -        // Language-specific scopes
Step #1: -        'cxx_common',
Step #1: -        'cxx_extractor',
Step #1: -        'cxx_indexer',
Step #1: -        'go_common',
Step #1: -        'go_extractor',
Step #1: -        'go_indexer',
Step #1: -        'java_common',
Step #1: -        'java_extractor',
Step #1: -        'java_indexer',
Step #1: -        'jvm_common',
Step #1: -        'jvm_extractor',
Step #1: -        'jvm_indexer',
Step #1: -        'rust_common',
Step #1: -        'rust_extractor',
Step #1: -        'rust_indexer',
Step #1: -        'typescript_common',
Step #1: -        'typescript_extractor',
Step #1: -        'typescript_indexer',
Step #1: -      ]
Step #1: -    ],
Step #1: -    'type-enum' : [
Step #1: -      2, 'always',
Step #1: -      [
Step #1: -        'build',
Step #1: -        'chore',
Step #1: -        'ci',
Step #1: -        'deprecation',
Step #1: -        'docs',
Step #1: -        'feat',
Step #1: -        'fix',
Step #1: -        'perf',
Step #1: -        'refactor',
Step #1: -        'release',
Step #1: -        'revert',
Step #1: -        'style',
Step #1: -        'test',
Step #1: -      ]
Step #1: -    ],
Step #1: +                     // Language-specific scopes
Step #1: +                     'cxx_common',
Step #1: +                     'cxx_extractor',
Step #1: +                     'cxx_indexer',
Step #1: +                     'go_common',
Step #1: +                     'go_extractor',
Step #1: +                     'go_indexer',
Step #1: +                     'java_common',
Step #1: +                     'java_extractor',
Step #1: +                     'java_indexer',
Step #1: +                     'jvm_common',
Step #1: +                     'jvm_extractor',
Step #1: +                     'jvm_indexer',
Step #1: +                     'rust_common',
Step #1: +                     'rust_extractor',
Step #1: +                     'rust_indexer',
Step #1: +                     'typescript_common',
Step #1: +                     'typescript_extractor',
Step #1: +                     'typescript_indexer',
Step #1: +                   ]
Step #1: +                 ],
Step #1: +    'type-enum' :
Step #1: +                [
Step #1: +                  2, 'always',
Step #1: +                  [
Step #1: +                    'build',
Step #1: +                    'chore',
Step #1: +                    'ci',
Step #1: +                    'deprecation',
Step #1: +                    'docs',
Step #1: +                    'feat',
Step #1: +                    'fix',
Step #1: +                    'perf',
Step #1: +                    'refactor',
Step #1: +                    'release',
Step #1: +                    'revert',
Step #1: +                    'style',
Step #1: +                    'test',
Step #1: +                  ]
Step #1: +                ],
Step #1:    },
Step #1:  }
Finished Step #1
```

**Comment by salguarnieri** [comment]
https://github.com/kythe/kythe/pull/5956 should make the format diff smaller

**Comment by shahms** [comment]
The `.commintlint.js` file is still unhappy, but I'll merge and fix it myself if the Presubmit passes.

**Review by shahms — APPROVED** [review]

**Comment by shahms** [comment]
Thanks!

**Pull request merged**
